### ISSUE TITLE
Use different styles for measure heading if description is available

### ISSUE THREAD

**Issue by georgf** [mozilla/telemetry-dashboard#406]
Comparing [a probe without description](https://mzl.la/2FxhYUN) and [a probe with description](https://mzl.la/2FwPive), you'll notice that one has a heading with the probe name, the other has a description.

What we should have instead is:
- always show the probe name with a "heading" style
- if there is a description, show a text section with it with a normal/regular font style

**Comment by Shruti9520** [comment]
Hi @georgf 
Can I work on this issue? :)

Thanks.

**Comment by georgf** [comment]
Sure, welcome!
Do you want to start digging into the code by yourself or would you like some pointers?

**Comment by Shruti9520** [comment]
I started looking into codebase. Some pointers would be appreciated. :)

**Comment by georgf** [comment]
#393 and #383 touch the same dashboard area, so this will potentially affect dashboards.css, dist.html, evo.html, evo.js, dist.js in `/new-pipeline`.
In Firefox, using right-click and "inspect element" should get started with where this is used in HTML.

**Comment by Shruti9520** [comment]
@georgf Thanks for the help. 
I have a doubt. I am sorry if I sound stupid.
When I run - `python -m http.server`, it gives me an error  "No module named http". So, I tried running ` python -m SimpleHTTPServer 8000`. But this doesn't help. Will you mind telling me the exact problem?

Thanks

**Comment by Shruti9520** [comment]
@georgf I tried running a remote webserver using `YOUR_USERNAME.github.io/telemetry-dashboard` after pushing changes to gh-pages branch. So it looks like this-
![mozilla_pr](https://user-images.githubusercontent.com/20228417/36812219-6adf465e-1cf6-11e8-8744-a52e5be44239.png)

Is that Okay?

**Comment by georgf** [comment]
Personally, for local testing i just:
- change my working directory to be inside the telemetry-dashboard directory
- `python -m SimpleHTTPServer 8000`
- in Firefox, navigate to `localhost:8000` or e.g. `localhost:8000/new-pipeline/dist.html`

Does that work for you? Do you run into any specific errors?

**Comment by georgf** [comment]
The screenshot looks like a great start!
The description text could use a smaller font size, probably just using the regular text size used elsewhere should be fine.

**Comment by Shruti9520** [comment]
@georgf It works fine. Thanks :)
Looking forward to submitting a PR.

**Comment by georgf** [comment]
Great, let us know if you need any further help!

**Comment by Shruti9520** [comment]
PR- [#427](https://github.com/mozilla/telemetry-dashboard/pull/427)
@georgf Thank you so much :)

**Comment by Sylvia23** [comment]
@chutten , Is this still open ?

**Comment by chutten** [comment]
I think @Shruti9520 is working on this in #472. Is that right?

**Comment by Sylvia23** [comment]
@Shruti9520 Can I take this if you want and not working on it? Thanks!

**Comment by shreya99oak** [comment]
@Shruti9520 I'll like to give it a shot too if it's open! since it seems @Sylvia23 is working on many other issues presently.

**Comment by Sylvia23** [comment]
@shreya99oak , It's follow up issue of the issue #383 which I solved earlier and touch the same part of the code. Also, I wrote code for this issue in that PR itself but @georgf created this as a separate issue hence told me to remove it from that PR. That's why, I wanted to take it. :) 
Regarding my other issues, I am working on two other issues right now which require in-depth knowledge of work-flow of code-base and discussions with the mentor unlike this one. 
So, I thought to give it a shot quickly.
